app/src/sharedTest/java/org/oppia/android/app/topic/TopicActivityTest.kt

```diff
@@ -164,6 +165,20 @@ class TopicActivityTest {
     assertThat(currentScreenNameWithIntentTwo).isEqualTo(ScreenName.TOPIC_ACTIVITY)
   }
 
+  @Test
+  fun testTopicActivity_createIntent_verifyProfileIdInIntent() {
+    val profileIdOne = TopicActivity.createTopicActivityIntent(
+      context, 1, FRACTIONS_TOPIC_ID
+    ).extractCurrentUserProfileId()
+
+    val profileIdTwo = TopicActivity.createTopicPlayStoryActivityIntent(
+      context, 1, FRACTIONS_TOPIC_ID, FRACTIONS_STORY_ID_0
+    ).extractCurrentUserProfileId()
+
+    assertThat(profileIdOne.internalId).isEqualTo(1)
+    assertThat(profileIdTwo.internalId).isEqualTo(1)
```

Comment:
What's the thought behind verifying two IDs that are actually the same here?

Reply:
Splitted this into two tests as they were previously verifying profileId in two different Activities.